src/elements/atoms/Divider/Divider.jsx

```diff
@@ -0,0 +1,69 @@
+/** Divider element is a dedicated component to give visual spacing between componentry. */
+
+export class Divider extends React.Component {
+  static propTypes = {
+    /** Tag overload */
+    tagName: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element,
+      PropTypes.func
+    ]),
+    /** Class stacking */
+    className: PropTypes.string,
+    /** Style variants */
+    variant: PropTypes.oneOf(['horizontal', 'vertical']),
+    /** Defines how wide it is across the screen */
+    length: PropTypes.oneOf(['large', 'medium', 'medium-small', 'small']),
+    /** Defines the thickness of the divider */
+    thickness: PropTypes.oneOf(['large', 'medium', 'small']),
+    /** Defines the color of the line */
+    color: PropTypes.oneOf(['black', 'gray', 'red', 'green']),
+    /** Children passed through */
+    children: PropTypes.node
+  };
+
+  static defaultProps = {
+    tagName: 'div',
```

Comment:
Lets use the <hr /> tag by default assuming this will mostly be used for a horizontal divide.
Then down in the render, if found to have a variant of `vartical` change Tag to be `div` instead.